package.json

```diff
@@ -89,7 +89,7 @@
   "devDependencies": {
     "@babel/core": "^7.11.1",
     "@mdn/dinocons": "^0.0.7",
-    "@mdn/minimalist": "^0.0.17",
+    "@mdn/minimalist": "^0.0.27",
```

Comment:
Perhaps we can set up Dependabot to have it make PRs against the `mdn-fiori` branch too. 
Having said that, I think it's healthy to often and periodically merge in `master` to keep the branch up-to-date too.

Reply:
Yup, I do that after each feature branch has been merged and before I create a new feature branch of `mdn-fiori`. The `minimalist` updates I do manually though as I need to add/change things there.